tools/pipelines/build-client.yml

```diff
@@ -99,6 +99,7 @@ extends:
     - ci:test:realsvc:local
     - ci:test:realsvc:tinylicious
     - ci:test:stress:tinylicious
+    - webpack
```

Comment:
Running `webpack` may changes the content of the package. It might be a good idea to example `build-npm-package.yml` and rearrange the order.

Reply:
I think if we consider webpack output to be part of a package's build (probably generally for example packages), we should instead make the build scripts reflect that. Perhaps a reasonable compromise could be to move `webpack` to be above `ci:test:mocha`. That does add some risk that the other test steps end up indirectly depending on the webpack step, but I think that risk is less realistic than the webpack step incorrectly modifying files it shouldn't in some build folder (and "corrupting" a package our CI has validated).